### ISSUE TITLE
When we go to sync ,we hit the start sync and we get sync failed so we do it over , and over until it finally works , it's multiple IPads , and more often than not does it do it we can have only one unit trying and we can have good connectivity, no matter

### ISSUE THREAD

**Issue by HATSBot** [WSDOT/HATS#460]
- **Device Name:** NCC6348047
- **Device Model:** iPad Air
- **Device System Version:** 9.3.2
- **Application Version:** HATS-iOS 1.5
- **Name:** Sam Schneider 
- **Has Attachments:** False
- **Feedback:** When we go to sync ,we hit the start sync and we get sync failed so we do it over , and over until it finally works , it&#39;s multiple IPads , and more often than not does it do it we can have only one unit trying and we can have good connectivity, no matter
- **Details:**  <a href='http://hatsprod.wsdot.loc/Maintenance/management/activities/Feedback/GetMobileDeviceFeedback?mobileFeedbackId=1891'>View Details</a>

**Comment by jhudsonWA** [comment]
Sam,

Do you have one of the error messages that show up when you do a sync?
- Josh

On Jun 1, 2016, at 5:57 AM, HATS Bot <notifications@github.com<mailto:notifications@github.com>> wrote:
-   Device Name: NCC6348047
-   Device Model: iPad Air
-   Device System Version: 9.3.2
-   Application Version: HATS-iOS 1.5
-   Name: Sam Schneider
-   Has Attachments: False
-   Feedback: When we go to sync ,we hit the start sync and we get sync failed so we do it over , and over until it finally works , it's multiple IPads , and more often than not does it do it we can have only one unit trying and we can have good connectivity, no matter
-   Details: View Detailshttp://hatsprod.wsdot.loc/Maintenance/management/activities/Feedback/GetMobileDeviceFeedback?mobileFeedbackId=1891

**Comment by jhudsonWA** [comment]
I looked into this. Last error was on 5/25: Most of the times this iPad at least seems to complete fine. I reached out to Sam but never heard back on other iPads. To keep the issue list small I'm going to close this one.

"Action: LogMessage • Device Name: NCC6348047 • Device Model: iPad Air • Device System Version: 9.3.2 • Application Version: HATS-iOS 1.5 • Message: Sync Complete With Errors • Log:

 05-25-2016 03:54:35 PM:
Log Sync Start Error: The Internet connection appears to be offline.

05-25-2016 03:54:35 PM:
1 Other Activity Record To Sync

05-25-2016 03:54:35 PM:
10 New Features To Sync

05-25-2016 03:54:34 PM:
Number of Days Since Last Sync: 1

05-25-2016 03:54:34 PM:
Starting Sync...

"
